pom.xml

```diff
@@ -131,5 +137,11 @@
             <version>2.13</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.jenkinsci.plugins</groupId>
+            <artifactId>pipeline-model-definition</artifactId>
+            <version>1.2.5</version>
+            <scope>test</scope>
```

Comment:
This was added a test-scoped dependency so that I could use a declarative Jenkinsfile in the test.